### ISSUE TITLE
Wiki is lacking since fbtft_device has been removed

### ISSUE THREAD

**Issue by muesli4** [notro/fbtft#554]
Since `fbtft_device` was removed I can no longer use my display. I tried to find a solution that involves device tree overlays but the documentation is so high level that I couldn't find anything useful in 2 hours. Before I could easily connect a display with custom pins. What is the proposed solution now?

**Comment by Sindbad6** [comment]
Yeah, same to me. Would be nice to explain in the Wiki, how to get it working in the device tree.
Thanks alot in advanced.

**Comment by Broodie** [comment]
Maybe you can try this out: https://github.com/Broodie/ili9341_tft_ubuntu_20_04
I've just managed to make an ili9341 display working.

**Comment by muesli4** [comment]
The issue is still present.

**Comment by notro** [comment]
I've updated the wiki and added an overlay that partially replaces fbtft_device: https://github.com/notro/fbtft/wiki/FBTFT-RPI-overlays

**Comment by muesli4** [comment]
@notro, I appreciate your effort.  However, in terms of usability I think this is devastating compared to the old module.  I do not even understand what half of that stuff does.  Now everyone that deviates a slight bit by not purchasing ready-made modules is forced to deal with this crap.  All I want to do is assign pins and display settings.  Where is the abstraction for details here?  This is frustrating and deters people from doing great stuff.

How would I convert from module arguments to a dts file?  I'm afraid a partial replacement is not sufficient.

**Comment by muesli4** [comment]
I was forced to do a system update and consequently had to read `/boot/overlays/README` again where I found the `fbtft` overlay.  This is specified in the wiki and if I would have paid more attention to the wall of text then I might have found it sooner.  In that sense, I would like to apologize for my last comment that I posted out of sheer frustration.

In the end the effort was minuscule and it was easy to simply adapt the parameters in the same way the kernel module received them.

If I may suggest some improvements for readability:
* Separate the wall of text into different sections. _I seem to not to be the only person that managed to simply read over it._
* Provide a simple example on how to use the overlay especially when transitioning from the kernel module.  _Examples bring clarity._
